src/mame/taito/qix.cpp

```diff
@@ -1370,7 +1403,7 @@ int qix_state::kram3_permut2(int tbl_index, int idx, const uint8_t *xor_table)
 	return xorval;
 }
 
-int qix_state::kram3_decrypt(int address, int value)
+static inline int decrypt(int address, int value)
```

Comment:
This is only relevant to the encrypted Kram game – it’s better the make the name reflect that.